tests/modules/services/emacs/emacs-socket-and-startWithUserSession.nix

```diff
@@ -0,0 +1,29 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+in {
+  config = {
```

Comment:
Can remove the `config = {` since no option is introduced:
```suggestion
```

Reply:
removed